includes/express-checkout/class-wc-payments-express-checkout-button-handler.php

```diff
@@ -0,0 +1,555 @@
+<?php
+/**
+ * Class WC_Payments_Express_Checkout_Button_Handler
+ * Adds support for Apple Pay, Google Pay and ECE API buttons.
+ * Utilizes the Stripe Express Checkout Element to support checkout from the product detail and cart pages.
+ *
+ * @package WooCommerce\Payments
+ */
+
+use WCPay\Exceptions\Invalid_Price_Exception;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use WCPay\Fraud_Prevention\Fraud_Prevention_Service;
+use WCPay\Logger;
```

Comment:
Suggestion to put all imports together.

```suggestion
if ( ! defined( 'ABSPATH' ) ) {
	exit;
}

use WCPay\Exceptions\Invalid_Price_Exception;
use WCPay\Fraud_Prevention\Fraud_Prevention_Service;
use WCPay\Logger;
```